### ISSUE TITLE
[Testing] Fix tooltip axe-core test and format a11y tests

### ISSUE THREAD

**Issue by a11ymiko** [sl-design-system/components#3535]
### Provide a general summary of the issue here

Right now a11y tests are scanning `sl-tootip` page when tooltip is not visible.
Some a11y tests are missing proper formating.

### 👤 Your name

Kacper

### 🧢 Your Product/Team

SLDS
